### PR TITLE
Add Flipper configuration

### PR DESCRIPTION
**Pull request by Lockyy**
### Context

There are various configuration options available for Flipper-UI. To make sure that the interface fits what we actually need this PR sets up the configuration.

### Changes proposed in this pull request

- Add confirmation box when clicking "fully enable" feature
- Turn off the recommendation to set up Flipper CLoud
- Disable ability to delete feature flags from the UI
- Disable ability to add feature flags from the UI
- Add link back to admin interface to UI

### CODE REVIEW

**Comment by Lockyy** [comment]
There is currently a bug in the Flipper-UI configuration options where if you have `confirm_fully_enable` on and `feature_removal_enabled` off then the "Fully enable" button does not work since the confirmation window doesn't properly compare the entered text against the feature name. 

I've filed a PR for the Flipper-UI gem to resolve this that will unblock this PR, see https://github.com/jnunemaker/flipper/pull/680.

**Review by peteryates — APPROVED** [review]
Looks great.

**Comment by Lockyy** [comment]
Fix has been merged into Flipper now, awaiting the next release with the fix before merging.

**Pull request merged**
